src/main/java/athena/AddCommand.java

```diff
@@ -0,0 +1,29 @@
+package athena;
+
+import java.io.IOException;
+
+/**
+ * Handles adding tasks to the TaskList.
+ */
+public class AddCommand extends Commands {
+    protected static String cmd;
+    protected static String details;
+
+    public AddCommand(String s, String d) {
```

Comment:
Consider renaming `s` and `d` to more descriptive names? Like `cmd` and `details`